src/backend/marsha/core/api/live_session.py

```diff
@@ -185,7 +185,7 @@ def partial_update(self, request, *args, **kwargs):
 
     @action(detail=False, methods=["post"])
     # pylint: disable=unused-argument
-    def push_attendance(self, request, pk=None):
+    def push_attendance(self, request, video_id=None):
```

Comment:
You don't plan to use it in this PR ?

Reply:
Nope, but it is required otherwise it raises an error (video_id not expected).